File: 2. Java/1. Bronze/10989.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class Main {
    public static void main(String[] args) throws IOException {
        // Variables
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int N, temp;
        int res[] = new int[10001];

        // Take Inputs
        N = Integer.parseInt(br.readLine());
        for (int i = 0; i < N; i++) {
            temp = Integer.parseInt(br.readLine());
            res[temp] += 1;
        }
        br.close();

        // Print Result
        for (int i = 0; i < 10001; i++) {
            for (int j = 0; j < res[i]; j++) sb.append(i).append('\n');
        }
        System.out.println(sb);
    }
}

File: 3. Python/1. Bronze/10989.py
import sys

# Variables
N = int(sys.stdin.readline())
res = {}

# Take Inputs
for i in range(N):
    temp = int(sys.stdin.readline())
    if temp in res: res[temp] += 1
    else: res[temp] = 1
    
# Print Result
for item in sorted(res.items()):
    for j in range(item[1]): print(item[0])